R/extremes-data.R

```diff
@@ -135,6 +135,31 @@ createDataRows <- function(data, param, rowName, isUpchain, includeRelated=TRUE,
         dataRows <- dataRows[!duplicated(dataRows[c("date", "primary")]),]
       }
 
+      #Keep only the row with the min/max corresponding value for a given day
+      if(includeRelated && doMerge){
+        if(!isUpchain){
```

Comment:
is there any way to combine the stuff in these two if statements with the ones on L132? Seems very similar, not sure if you've looked into that yet and it just didn't work.